docs/base-guide/index.md

```diff
@@ -4,9 +4,9 @@ hide:
 ---
 
 # The Discordoo guide
```

Comment:
```suggestion
# The Discordoo Guide
```

Reply:
this is a title and thus everything has to be capitalized, e.g. New York, but not New york